config/setup/errors.go

```diff
@@ -21,12 +23,17 @@ func Errors(c *Controller) (middleware.Middleware, error) {
 	// Open the log file for writing when the server starts
 	c.Startup = append(c.Startup, func() error {
 		var err error
-		var file *os.File
+		var file io.Writer
 
 		if handler.LogFile == "stdout" {
 			file = os.Stdout
 		} else if handler.LogFile == "stderr" {
 			file = os.Stderr
+		} else if handler.LogFile == "syslog" {
+			file, err = gsyslog.NewLogger(gsyslog.LOG_ERR, "SYSLOG", "caddy")
```

Comment:
It does make sense, I didn't put too much thoughts into that, but I should have.
As for providing the LOCAL number, it could be a good idea, but the more parameters you put, the more complicated the configuration file become. I guess we could accept just `syslog` as well and use a default one in that case. I am curious to know what @mholt thinks about it.